Makefile

```diff
@@ -24,7 +24,7 @@ endif
 ifeq ($(PAASTA_ENV),YELP)
 	export DOCKER_REGISTRY ?= docker-dev.yelpcorp.com/
 else
-	export DOCKER_REGISTRY ?= ""
+	export DOCKER_REGISTRY ?=
```

Comment:
This could well have some unintentional consequences... imo you can probably ship this if tests pass and if it works on normal public builds (unless @nemacysts already recognizes a problem here) 👍 

If not, we may have to look a bit closer, but a quick fix would be to set it back to "" and just modify the codegen docker cmds as we have done elsewhere: https://github.com/Yelp/paasta/blob/master/yelp_package/Makefile#L38

Reply:
i *think* this would work? we could also probably sidestep this "" vs empty string issue by setting the `else` case to the full dockerhub url like the original commit did :)